Source/WebCore/platform/graphics/freetype/FontCacheFreeType.cpp

```diff
@@ -239,16 +239,17 @@ struct FallbackFontDescriptionKeyHash {
     static const bool safeToCompareToEmptyOrDeleted = true;
 };
 
-using SystemFallbackCache = HashMap<FallbackFontDescriptionKey, std::unique_ptr<CachedFontSet>, FallbackFontDescriptionKeyHash, SimpleClassHashTraits<FallbackFontDescriptionKey>>;
-static SystemFallbackCache& systemFallbackCache()
+using SystemFallbackFontSetCache = HashMap<FallbackFontDescriptionKey, std::unique_ptr<CachedFontSet>, FallbackFontDescriptionKeyHash, SimpleClassHashTraits<FallbackFontDescriptionKey>>;
+static SystemFallbackFontSetCache& systemFallbackFontSetCache()
 {
-    static NeverDestroyed<SystemFallbackCache> cache;
+    // FIXME: This should be moved to FontCache since it can be accessed from worker threads.
```

Comment:
Hm, I think _none_ of the font code is threadsafe. @carlosgcampos, do you have time to look into this?